src/lib/ildcp.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+const IlpPacket = require('ilp-packet')
+const compat = require('ilp-compat-plugin')
+const { Reader } = require('oer-utils')
+const debug = require('debug')('ilp:ildcp')
+
+const PEER_PROTOCOL_CONDITION = Buffer.from('Zmh6rfhivXdsj8GLjp+OIAiXFIVu4jOzkCpZHQ1fKSU=', 'base64')
+const PEER_PROTOCOL_EXPIRY_DURATION = 60000
+
+const getAccount = async (plugin) => {
+  plugin = compat(plugin)
+
+  const data = await plugin.sendData(IlpPacket.serializeIlpPrepare({
+    // TODO: Should be zero, but our plugins suck
```

Comment:
lol

...we should probably make them not suck because the connector will actually fulfill this

Reply:
Hehe, this comment was from a previous version. No longer applicable because in ilp4 the plugin doesn't even know the amount. (sendData just sends opaque data and for amount=0 you just don't sendMoney afterwards.)